src/validators/duration_distance.rs

```diff
@@ -49,20 +49,41 @@ fn validate_speeds(gtfs: &gtfs_structures::Gtfs) -> Result<Vec<Issue>, gtfs_stru
         let route = gtfs.get_route(&trip.route_id)?;
         for (departure, arrival) in trip.stop_times.iter().tuple_windows() {
             if let Some((distance, duration)) = distance_and_duration(departure, arrival) {
-                let issue_kind = if distance < 10.0 {
-                    Some((Severity::Information, IssueType::CloseStops))
+                let (issue_kind, details) = if distance < 10.0 {
+                    (
+                        Some((Severity::Information, IssueType::CloseStops)),
```

Comment:
hum wouldn't it be easier to return an Option with a bigger tuple instead of 2 options ?
(`Some(Severity::Information, IssueType::CloseStops, format!( between the stops is {:.0} meter(s)", distance))`